_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,365 @@
+  development and a proposed watchtower protocol specification. He
+  explored the various trade-offs when designing a watchtower and the
+  interplay between privacy requirements, accessibility, storage, and
+  fees charged.  Delgado is working on the watchtower implementation
+  [The Eye of Satoshi][teos] at Talaia Labs, which is aiming to be
+  compliant with the proposed BOLT13 (see [Newsletter #75][news75
+  watchtower bolt]).  Delgado also highlighted how watchtowers are
+  becoming increasingly critical in multiple settings such as Bitcoin
+  [vault][topic vaults] designs, statechains, and atomic swaps in addition to LN.
+  ([transcript][watchtowers ts], [video][watchtowers vid],
+  [slides][watchtowers slides])
+
+- **CoinSwap:** Aviv Milner presented on CoinSwap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  CoinSwap proposal (see [Newsletter #100][news100 coinswap]) provides
+  covertness in a manner that other privacy schemes such as
+  [coinjoin][topic coinjoin] fail to do. Milner also went through the
+  motivation for routing CoinSwaps, namely to address the concern of
+  unwittingly entering into a CoinSwap with an adversary such a chain
+  surveillance company.  ([transcript][coinswap ts], [video][coinswap
+  vid])
+
+- **Taproot and schnorr multisignatures:** Some of the co-authors of [BIP340][],
+  including Tim Ruffing, Pieter Wuille, and Jonas Nick, held a wide ranging
+  discussion at London Bitcoin Devs on the specification of [schnorr
+  signatures][topic schnorr signatures].  This covered some of the
+  earlier ideas for implementing schnorr in Bitcoin and what the
+  co-authors thought the community should be concerned about when
+  considering a possible future soft fork deployment. Wuille expressed
+  his view that it is unlikely at this stage that there are bugs in the
+  proposed consensus implementation and that he is more concerned with
+  how schnorr is adopted and what is built using it.
+  ([transcript][socratic ts], [video][socratic vid])
+
+    The following day, Tim Ruffing presented on multisignature and
+    threshold signature schemes using schnorr signatures. He highlighted
+    how the rogue key attack and an attack using Wagner’s algorithm (see
+    [Newsletter #74][news74 wagner]) need to be addressed when designing
+    secure multisignature schemes.  Ruffing has been working with
+    collaborators on a speculative [MuSig][topic musig] signature scheme
+    that only requires two rounds of interaction rather than three
+    (without relying on zero knowledge proofs). Threshold signature
+    schemes generally present further challenges such as requiring an
+    honest majority, a broadcast channel, and careful consideration of
+    parallel signing sessions.  ([transcript][ruffing ts],
+    [video][ruffing vid], [slides][ruffing slides])
+
+
+- **Sydney meetup discussion:** A number of Bitcoin developers and LN
```

Comment:
I suggest we limit this section to 4 transcripts. If you agree, I think we should drop this one, since it's the least focused of the 5 talks.

Reply:
Not my call to make but let me know for next month what the requirements are please. Always open to feedback on whether it is too long/too short/not of interest to Optech readers etc. I certainly thought there should be a longer section on the Schnorr multisig stuff this month.

Reply:
In our initial conversations about the section, I told Michael, "I'd prefer the section be no more than 5 transcripts and a maximum of 700 words".   We're currently at 682 words for the section (including link anchors), so if >4 transcripts is a problem, I want to be clear that's my fault.

I think maybe this section looks so long because it has two items that each have two paragraphs.  Maybe instead of a 4 or 5 item limit, we have a 5 paragraph total limit?  Also, we could lower the maximum word count limit a bit to make the section shorter no matter what the item/paragraph limit is.

I'll make the other edits you suggest in one commit, and then---in a separate commit---I'll see if I can't compress this down to just five paragraphs without losing too much information (or creating crazy long paragraphs).  Would that be a reasonable compromise for this month at least?